trunk/NDHMS/Routing/Reservoirs/module_read_timeslice_data.F

```diff
@@ -0,0 +1,354 @@
+! This module reads USGS timeslice files to get gage discharge
```

Comment:
Sort of thinking this module name should be 
"module_reservoir_read_timeslice_data". 
the name seems to generic. 
I dont have a strong opinion...

Reply:
at the file system level it's clear, but at the modle `use` level it's less clearn

Reply:
@jdmattern-noaa fixing. Changing to "module_reservoir_read_timeslice_data".